### PR TITLE
Typo + Tuebingen Configs

### PR DESCRIPTION
**Pull request by apeltzer**
- Found a small typo
- Added our Tuebingen cluster configs

### CODE REVIEW

File: conf/binac.config [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+singularity {
+    enabled = true
+}
+
+process {
+    beforeScript = 'module load devel/singularity/2.4.1'
+    executor = 'pbs'
+    container = 'docker://scilifelab/ngi-methylseq'
```
Comment (ewels) [inline conf/binac.config]: Sorry, just noticed this - probably the same on the RNA-seq pipeline. Quite recently I added a new small feature to automatically use the tagged version of the docker container if running a tagged version of the pipeline. Relevant code [here](https://github.com/SciLifeLab/NGI-MethylSeq/blob/master/nextflow.config#L13-L16) and use in a config file [here](https://github.com/SciLifeLab/NGI-MethylSeq/blob/master/conf/uppmax.config#L18).

Comment (ewels) [reply conf/binac.config]: To clarify - just switch this line to this:

```
container = "docker://$wf_container"
```

File: conf/cfc.config [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+*To be improved by process specific configuration asap, once our CFC cluster has the extra options removed - till then, task.attempt in NextFlow is not supported there.
+*/
+
+process {
+    cpus = 20
+    time = 48.h
+    memory = 60.GB
+    beforeScript = 'module load qbic/singularity/2.4'
+    executor = 'pbs'
+    container = 'docker://scilifelab/ngi-methylseq'
```
Comment (ewels) [inline conf/cfc.config]: Switch to:

```
container = "docker://$wf_container"
```

**Comment by apeltzer** [comment]
Hi Phil! Thanks for the comment - I added this and changed according to your request.

File: conf/binac.config [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ */
+
+singularity {
+    enabled = true
+}
+
+process {
+    beforeScript = 'module load devel/singularity/2.4.1'
+    executor = 'pbs'
+    container = 'docker://$wf_container'
```
Comment (ewels) [inline conf/binac.config]: Hmm, you may need double quotes here. Also we may need to use `{}` to defer execution? Difficult to actually test this until we have more tagged releases available!

Comment (ewels) [reply conf/binac.config]: Hi @apeltzer - does this work as it stands? Let me know if you'd like me to merge anyway.

Comment (apeltzer) [reply conf/binac.config]: Oh, sorry. I corrected this in NGI-RNAseq, but forgot to do so here. I'll change it and send again.

Comment (apeltzer) [reply conf/binac.config]: There you go!

**Comment by ewels** [comment]
Great, thanks! Tests are failing because our hosting for the test data has gone down. We're working on replacing this.

Phil

**Pull request merged**